plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExMemoryUsage.hs

```diff
@@ -258,14 +258,35 @@ instance ExMemoryUsage Data where
             I n        -> memoryUsage n
             B b        -> memoryUsage b
 
+
```

Comment:
This is in response to [this comment](https://github.com/input-output-hk/plutus/pull/5231#discussion_r1192712631) from @effectfully .